projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v3-endpoint-blogging-prompts.php

```diff
@@ -533,11 +537,11 @@ public function proxy_request_to_wpcom( $request, $path = '' ) {
 		}
 
 		$response_status = wp_remote_retrieve_response_code( $response );
-		$response_body   = json_decode( wp_remote_retrieve_body( $response ) );
+		$response_body   = json_decode( wp_remote_retrieve_body( $response ), true );
```

Comment:
this was the fix so that an array type is returned on both wpcom and self hosted